src/main/kotlin/br/unipar/plano/domain/reembolso/model/Carteirinha.kt

```diff
@@ -0,0 +1,27 @@
+package br.unipar.plano.domain.reembolso.model
+
+import java.time.LocalDate
+import java.util.*
+import javax.persistence.*
+
+enum class StatusCarteirinha {
+    ATIVA, VENCIDA
+}
+
+@Entity
+class Carteirinha (
+    @Id
+    val id: UUID,
```

Comment:
O ideal seria um value object para esse tipo de dados

```suggestion
    val id: IdCarteirinha,
```